mcp/src/minecraft/net/minecraft/block/Block.java

```diff
@@ -1636,14 +1636,16 @@ public static void registerBlocks()
         registerBlock(783, "diagonal_bricks", (new Block(Material.ROCK)).setHardness(1.5F).setResistance(10.0F).setSoundType(SoundType.STONE).setUnlocalizedName("diagonalBricks").setCreativeTab(CreativeTabs.BUILDING_BLOCKS));
         //registerBlock(784, "dark_brown_wood", (new BlockValarDirectional(Material.wood)).setHardness(2.0F).setSoundType(soundTypeWood).setUnlocalizedName("darkBrownWood").setBlockTextureName("dark_brown_wood"));
         registerBlock(785, "bell", (new BlockBell()).setUnlocalizedName("bell"));
-        registerBlock(786, "elven_crafting_table", (new BlockElvenWorkbench()).setHardness(2.5F).setSoundType(SoundType.WOOD).setUnlocalizedName("elvenCraftingTable"));
-        //787 - human_crafting_table
-        //788 - gondorial_crafting_table
-        //789 - rohhirim_crafting_table
-        //790 - hobbit_crafting_table
-        //791 - mordor_crafting_table
-        //792 - isengard_crafting_table
-        //793 - goblin_crafting_table
+        Block block500 = (new BlockCustomWorkbench()).setSoundType(SoundType.WOOD).setHardness(0.8F).setUnlocalizedName("customWorkbench");
```

Comment:
All vanilla blocks which have subblocks used this :smile: but it doesnt seem to make any difference...